### PR TITLE
set the image to resize the canvas on load, and add jpeg support

### PR DESCRIPTION
**Pull request by steveoc64**
Use the updated ImageFillResize Fillmode on loaded images.

Also -when loading into a temp file, tag the temp file with the suffix jpg / png ... not that it matters too much, but makes debugging a bit easier.

### CODE REVIEW

**Comment by steveoc64** [comment]
requires https://github.com/fyne-io/fyne/pull/50

File: xkcd/main.go [hunk 1]
```diff
@@ -155,8 +159,8 @@ func Show(app fyne.App) {
 	w := app.NewWindow("XKCD Viewer")
 
 	form := x.NewForm(w)
-	x.image = &canvas.Image{}
```
Comment (andydotxyz) [inline xkcd/main.go:158]: This portion has been implemented. Do you want to update to apply the other code or close and open a new one?

**Comment by andydotxyz** [comment]
Any update on this patch?

**Comment by steveoc64** [comment]
Last I looked, the image sizing was working well with the examples, so may have been resolved (I think as part of what Lucor did with the form widget ??)

There was also the issue of getting canvas widgets to be able to report on the raw image size, which was one solution.

Ill close the PR now then, and wait till an actual issue gets posted - if it needs any extra work at all.

**Pull request closed without merging**